sycl/doc/extensions/experimental/sycl_ext_oneapi_graph.asciidoc

```diff
@@ -313,6 +313,15 @@ enum class graph_support_level {
   emulated
 };
 
+enum class graph_debug_print_flag {
```

Comment:
If these are flags, is the intention that a user can bitwise or them?

Reply:
Yes it is. To let themselves define how verbose they want the printed graph to be.

Reply:
If it try this in compiler explorer 
```
enum class graph_debug_print_flag {
  none = 0x00,
  kernel_args = 0x01,        // adds the printing of the kernel args
                             // where applicable
  memory_access_desc = 0x02  // adds the printing of information related to
                             // the memory access: source address,
                             // destination address, length, etc.
};

int main() {
  graph_debug_print_flag foo = graph_debug_print_flag::none | graph_debug_print_flag::kernel_args;

}
```
I get an error because it's a c++ enum class
```<source>:11:61: error: invalid operands to binary expression ('graph_debug_print_flag' and 'graph_debug_print_flag')
   11 |   graph_debug_print_flag foo = graph_debug_print_flag::none | graph_debug_print_flag::kernel_args;
      |             
```

Reply:
Yes, you are right. I updated definition of this flag using bit-field (mimicking how flags are defined in the PI layer) .

Reply:
These bitfields look like something from a C-API, could we just make the second parameter to `print_dot_graph()`  a bool for verbose print? `  void print_dot_graph(std::string path, bool verbose = false) const;` Then leave the exact details of what's in that verbose printing unspecified. Not sure if printing both kernel-args and memory-accesses if `verbose` is set is too noisy though.

Reply:
Okay. I updated the specs.